### PR TITLE
add Leaderboard UI [#2]

### PR DESCRIPTION
**Pull request by Drapegnik**
- https://lzrby.github.io/squash

Screen Shot 2019-05-08 at 20 02 40

### CODE REVIEW

File: leaderboard-ui/src/raiting.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "name": "klicunou",
+        "image": "https://scontent-frt3-2.xx.fbcdn.net/v/t1.0-1/c0.0.320.320a/p320x320/44827977_856027851454515_3991474135086661632_n.jpg?_nc_cat=107&_nc_ht=scontent-frt3-2.xx&oh=82af57d3bdaf47e98e04d5acd181eb95&oe=5D628F9C",
```
Comment (KlicOgogo) [inline leaderboard-ui/src/raiting.json]: Please explain what will happen if I change my facebook user picture.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,11 @@
 - `--ball="<ball color>"` flag is used when play ball was red or blue. Yellow color is default.
 - Also see `./logger.py --help`.
 
-### Get ratings
+### get ratings
 
 - Run from terminal at repo root: `./get_ratings.py --day=<yyyy-mm-dd>`. This scripts prints current ratings for day from input at 00:00.
 - Also you can see `./get_ratings.py --help`.
+
+### ui
```
Comment (KlicOgogo) [inline README.md:24]: Do we need any installation guide?

File: leaderboard-ui/src/raiting.json (hunk 1 above)
Comment (Drapegnik) [reply leaderboard-ui/src/raiting.json]: @KlicOgogo, yeah, this is temp solution, we should save and store photos here

File: README.md (hunk 2 above)
Comment (Drapegnik) [reply README.md:24]: `npm i`

and `npm` comes with `node`, I suggest install it via [`nvm`](https://github.com/nvm-sh/nvm)

**Review by KlicOgogo — APPROVED** [review]

File: leaderboard-ui/src/raiting.json (hunk 1 above)
Comment (KlicOgogo) [reply leaderboard-ui/src/raiting.json]: OK

File: README.md (hunk 2 above)
Comment (KlicOgogo) [reply README.md:24]: I will try that after merge

**Pull request merged**